packages/gutenpride-template/package.json

```diff
@@ -0,0 +1,20 @@
+{
+	"name": "@wordpress/gutenpride-template",
+	"version": "1.0.0-prerelease",
+	"description": "A package containing a block template to use with @wordpress/create-block.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [ "wordpress", "create block", "gutenpride", "block template" ],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/gutenpride-template/README.md",
```

Comment:
Should we reference the tutorial here instead?

Reply:
I referenced the tutorial link here.